spec/lib/telephony/pinpoint/sms_sender_spec.rb

```diff
@@ -471,13 +447,19 @@ def ==(other)
     end
   end
 
-  def mock_build_client(client = mock_client)
-    Telephony::Pinpoint::SmsSender::CLIENT_POOL[sms_config] = FakeConnectionPool.new { client }
-  end
+  def mock_build_clients(client = mock_client, backup_client = backup_mock_client)
```

Comment:
can we make these keyword arguments? I was confused when the callsites all had `(mock_client, nil)`
```suggestion
  def mock_build_clients(client: mock_client, backup_client: backup_mock_client)
```

Reply:
done!